### ISSUE TITLE
j2lint prints error messages twice in log file after running with --log

### ISSUE THREAD

**Issue by KonikaChaurasiya-GSLab** [aristanetworks/j2lint#26]
```2022-02-24 17:31:12,417 - root - INFO - Created collection from rules directory ~/Documents/j2lintEnv/lib/python3.8/site-packages/j2lint-0.1-py3.8.egg/j2lint/rules
2022-02-24 17:31:12,426 - root - ERROR - ~/Documents/template.j2:6 All variables shall use lower case: '{{ variable }}' (jinja-variable-lower-case)
2022-02-24 17:31:12,426 - root - INFO - Linting errors found [~/Documents/template.j2:6 All variables shall use lower case: '{{ variable }}' (jinja-variable-lower-case)]
2022-02-24 17:31:12,426 - root - ERROR - ~/Documents/template.j2:6 All variables shall use lower case: '{{ variable }}' (jinja-variable-lower-case)```

**Comment by Shivani-chourasiya** [comment]
Resolved in PR #44

**Comment by Shivani-chourasiya** [comment]
Resolved in PR #44
